test/components/FeatureFlagRenderer.test.js

```diff
@@ -164,28 +172,54 @@ describe("components/FeatureFlagRenderer", () => {
 
     context("when the feature flag comes back true", () => {
       it("sets the state", () => {
-        const wrapper = mount(
-          <FeatureFlagRenderer
-            launchDarklyConfig={launchDarklyConfig}
-            flagKey="my-test"
-            renderFeatureCallback={renderFeatureCallback}
-          />
-        );
+        variation.returns(true);
+        const wrapper = getWrapper();
         expect(wrapper.state()).to.deep.equal({ checkFeatureFlagComplete: true, showFeature: true });
       });
     });
 
     context("when the feature flag comes back false", () => {
       it("sets the state", () => {
-        const wrapper = mount(
-          <FeatureFlagRenderer
-            launchDarklyConfig={launchDarklyConfig}
-            flagKey="my-test"
-            renderFeatureCallback={renderFeatureCallback}
-          />
-        );
+        variation.returns(false);
+        const wrapper = getWrapper();
+        expect(wrapper.state()).to.deep.equal({ checkFeatureFlagComplete: true, showFeature: false });
+      });
+    });
+
+    describe("query param flag overrides if not undefined", () => {
+      let _sandbox;
+      before(() => {
+        _sandbox = sandbox.create();
+      });
+      afterEach(() => {
+        _sandbox.restore();
+      })
+      it("param \"features.flag=false\" overrides LD data \"on\"", () => {
```

Comment:
Minor nitpick but instead of escaping the double quotes here, lets just use single quotes or backticks.